assets/src/edit-story/utils/useClipboardHandlers.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+const BLACKLIST_CLIPBOARD_ELEMENTS = [ 'INPUT', 'TEXTAREA' ];
+
+/**
+ * @param {?Element} container
+ * @param {function(!ClipboardEvent)} copyCutHandler
+ * @param {function(!ClipboardEvent)} pasteHandler
+ */
+function useClipboardHandlers( container, copyCutHandler, pasteHandler ) {
+	useEffect( () => {
+		if ( ! container ) {
+			return undefined;
+		}
+
+		const copyCutHandlerWrapper = ( evt ) => {
+			const { target, clipboardData } = evt;
+
+			// Elements that either handle their own clipboard or use platform.
+			if ( ! target ||
+          BLACKLIST_CLIPBOARD_ELEMENTS.includes( target.tagName ) ||
+          target.closest( '[contenteditable="true"]' ) ) {
+				return;
+			}
+
+			// A target can be anywhere in the container's full subtree, but not
+			// in its siblings.
+			if ( ! container.contains( target ) && ! target.contains( container ) ) {
+				return;
+			}
+
+			// Someone has already put something in the clipboard. Do not override.
+			if ( clipboardData.types.length !== 0 ) {
```

Comment:
Curious, in which case shouldn't it override what's already in the clipboard?

Reply:
TBH this might be overprotection. Just something I've done before, but don't remember any special problematic case.